modules/google-maps/src/utils.ts

```diff
@@ -142,7 +155,8 @@ export function getViewPropsFromOverlay(map, overlay) {
   // Maps sometimes returns undefined instead of 0
   const heading = map.getHeading() || 0;
 
-  let zoom = map.getZoom() - 1;
+  let zoom = (map.getZoom() ?? 0) - 1;
```

Comment:
@felixpalmer 
I'm not sure how act here. `getZoom` can return `undefined` and that then?

(see also previous comment)

Reply:
`getZoom` should only return undefined if the map hasn't been initialized yet. As the code above (see my other comment) will catch this case when accessing `getBounds` I would say we should assume `getZoom` always returns a valid value and use `!`.